### ISSUE TITLE
WGPU 0.15 compatibility

### ISSUE THREAD

**Issue by David-OConnor** [hasenbanck/egui_wgpu_backend#70]
```rust
181  |         let rpass_egui = RenderPass::new(device, surface_cfg.format, 1);
     |                          ^^^^^^^^^^^^^^^ ------  ------------------ expected enum `egui_wgpu_backend::wgpu::TextureFormat`, found enum `wgpu::TextureFormat`
     |                                          |
```
How should we tackle this? Thank you.

**Comment by hasenbanck** [comment]
This crate needs to be patched to support WGPU 0.15. PRs are welcome.

**Comment by hasenbanck** [comment]
We support wgpu 0.16 now.
